microsoft/testsuites/performance/common.py

```diff
@@ -510,6 +518,8 @@ def perf_sockperf(
     client = cast(RemoteNode, environment.nodes[0])
     server = cast(RemoteNode, environment.nodes[1])
     sysctls: List[Sysctl] = []
+    if isinstance(client.os, Ubuntu) and (client.os.information.version < "18.4.0"):
+        raise SkippedException
```

Comment:
please add detailed skip reason

Reply:
@mcgov please address this in a new PR, I will merge it since there are some failures caused by this case every day.